src/convertToObject.js

```diff
@@ -3,15 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
+  const styleCommand = sourceString.split(';');
+  const result = {};
+
+  for (const part of styleCommand) {
+    let twoPartsCommand = part.split(':');
+
+    if (twoPartsCommand[1]) {
+      result[twoPartsCommand[0].trimEnd().trimStart()]
+      = twoPartsCommand[1].trimEnd().trimStart();
+      twoPartsCommand = [];
```

Comment:
looks redundant

Reply:
<img width="474" alt="image" src="https://github.com/mate-academy/js_style-to-object/assets/70220667/4d5cabaf-a97d-4c63-870d-912ce3a6ab4b">